code/controllers/Processes/modifier_process.dm

```diff
@@ -0,0 +1,28 @@
+#define MODIFIER_PROCESS_INTERVAL	10
+
+/datum/controller/process/modifier/setup()
+	name = "modifiers"
+	schedule_interval = MODIFIER_PROCESS_INTERVAL // every 2 seconds
+	start_delay = 8
+
+/datum/controller/process/modifier/started()
+	..()
+	if(!processing_modifiers)
+		processing_modifiers = list()
+
+/datum/controller/process/modifier/doWork()
+	for(last_object in processing_modifiers)
+		var/datum/modifier/O = last_object
+		if(isnull(O.gcDestroyed))
+			try
+				O:process()
```

Comment:
I figured this was done to prevent one bad modifier from crashing all of them and breaking the system, a robustness measure.

not necessary im sure, done